pkg/provider/aws/persist.go

```diff
@@ -9,19 +9,19 @@ func (a *AWS) GenerateFsResources(construct *core.Fs, result *core.ConstructGrap
 	bucket := resources.NewS3Bucket(construct, a.Config.AppName)
 	dag.AddResource(bucket)
 	a.MapResourceDirectlyToConstruct(bucket, construct)
+	actions := []string{"s3:*"}
```

Comment:
Did this move so that we don't create multiple of the same bucket, in multi-exec? And did you catch it via any tests, or just inspection?

Reply:
before we were creating the policy for every single exec unit, so i moved it out so we create the policy once and attach it to the exec units roles that need it